content/master/concepts/compositions.md

```diff
@@ -664,40 +664,59 @@ spec:
 The [EnvironmentConfigs]({{<ref "./environment-configs" >}}) page has 
 more information on EnvironmentConfigs options and usage.
 
-### Composition functions
+### Use Composition Functions
```

Comment:
```
 667:5   warning  'Use Composition Functions'     Google.Headings
                  should use sentence-style
                  capitalization.
```

I'm feeling a little all over the place WRT whether I should capitalize Composition Functions, here in the heading and elsewhere. Some context that may help figure out the right pattern:

* Composition is a type. I couldn't find a clear pattern for when we say Composition vs composition in the docs.
* Function is a type (of package).
* I'd like to call Composition Functions just Functions" (or functions?) for short. Not xfns or XFNs.

Reply:
I mention this because if I'm following https://developers.google.com/style/headings correctly I think Vale would accept "Use composition functions" here as the heading.

Reply:
We've generally tried to follow the [Kubernetes docs guidelines](https://kubernetes.io/docs/contribute/style/style-guide/) on this. 

My approach is if I'm talking about a concept I use sentence case. I (try) and only capitalize when referring to the type, for example "Creating composite resource definitions" vs "Create a CompositeResourceDefinition object"

For titles, _most_ of the time I'd recommend sentence case with lowercase `functions` which may require rewording like "Use a composition function"